as/cNFT/assembly/core.ts

```diff
@@ -35,4 +37,9 @@ export function nft_transfer(token_id: TokenId, bidder_id: AccountId): void {
     /* Deleting token from previous owner */
     persistent_tokens.remove(token.id, token.prev_owner_id)
 
+    
+    // Immiting log event
+    const transfer_log = new NftTransferLog()
```

Comment:
`NftTransferLog` data should be added here. Like `old_owner_id`, `new_owner_id`, `tokens` etc ..

Reply:
Oh yes, I forgot that. Thanks for pointing it out!